powertools-parameters/src/main/java/software/amazon/lambda/powertools/parameters/cache/DataStore.java

```diff
@@ -46,7 +46,8 @@ public void remove(String Key){
     }
 
     public Object get(String key) {
-        return store.containsKey(key)?store.get(key).value:null;
+        ValueNode node = store.get(key);
+        return node ? node.value : null;
```

Comment:
@XinyuLiu5566  - this would fix the runtime error

```suggestion
        return node != null ? node.value : null;
```